src/main/java/com/epam/ta/reportportal/dao/LogRepositoryCustomImpl.java

```diff
@@ -123,25 +123,23 @@ public boolean hasLogs(Long itemId) {
 	}
 
 	@Override
-	public List<Long> findIdsByLaunchId(Long launchId) {
+	public List<Long> findItemLogIdsByLaunchId(Long launchId) {
 		return dsl.select()
 				.from(LOG)
 				.leftJoin(TEST_ITEM)
 				.onKey()
```

Comment:
let's make this in common style with 'launch join' below

Reply:
done